ProjectManager/ProjectManager/Controller/IssueViewController.swift

```diff
@@ -0,0 +1,182 @@
+    }()
+    
+    private var titleTextField: PaddedTextField = {
+        let padding = UIEdgeInsets(top: LayoutConstant.titleTextFieldPadding,
+                                   left: LayoutConstant.titleTextFieldPadding,
+                                   bottom: LayoutConstant.titleTextFieldPadding,
+                                   right: LayoutConstant.titleTextFieldPadding)
+        let textField = PaddedTextField(padding: padding)
+        textField.backgroundColor = .systemBackground
+        textField.placeholder = Namespace.title
+        textField.setContentHuggingPriority(.required, for: .vertical)
+        textField.addShadow(radius: LayoutConstant.shadowRadius)
+        
+        return textField
+    }()
+    
+    private var datePicker: UIDatePicker = {
+        let picker = UIDatePicker()
+        picker.datePickerMode = .date
+        picker.preferredDatePickerStyle = .wheels
+        picker.locale = .current
+        picker.translatesAutoresizingMaskIntoConstraints = false
+        
+        return picker
+    }()
+    
+    private var bodyTextView: UITextView = {
+        let textView = UITextView()
+        textView.backgroundColor = .systemBackground
+        textView.setContentHuggingPriority(.defaultLow, for: .vertical)
+        textView.addShadow(radius: LayoutConstant.shadowRadius)
+        
+        return textView
+    }()
+    
+    init(delegate: IssueManageable) {
+        self.delegate = delegate
+        super.init(nibName: nil, bundle: nil)
+        configureNavigationBar()
+    }
+    
+    init(issue: Issue, delegate: IssueManageable) {
+        self.issue = issue
+        self.delegate = delegate
+        defer { self.isEditable = false }
+        super.init(nibName: nil, bundle: nil)
+        configureNavigationBar()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        configureContents()
+    }
+    
+    private func configureUI() {
+        view.addSubview(stackView)
+        view.backgroundColor = .systemBackground
+        configureStackView()
+    }
+    
+    private func configureNavigationBar() {
+        if issue == nil {
+            title = Status.todo.description
+            navigationItem.rightBarButtonItem = UIBarButtonItem(title: Namespace.done,
+                                                                primaryAction: UIAction { _ in
+                self.createIssue()
+                self.delegate.addIssue(issue: self.issue)
+                self.dismiss(animated: true)
+            })
+            navigationItem.leftBarButtonItem = UIBarButtonItem(title: Namespace.cancel,
+                                                               primaryAction: UIAction { _ in
+                self.dismiss(animated: true)
+            })
+        } else {
+            title = issue?.status.description
```

Comment:
issue가 nil 아닐 때인데 차라리 조건을 

if let ~ 으로 처리해줘서 issue != nil 일 때는 issue가 옵셔널로 처리되지 않도록 수정해주는 건 어떨까요?
```suggestion
            title = issue.status.description
```

Reply:
혀너블이 조언해주신 대로 고쳐보려고 했는데, `IssueViewController`가 가지고 있는 `issue` 프로퍼티의 값을 업데이트 해주는 함수가 조건문 안에서 호출되고 있어서
조건문 밖에서 바인딩을 할 경우 업데이트 되지 않은 `issue`를 전달하지 않는 문제가 발생했습니다 😥

> issue를 optional로 받지 않고 해당 메서드를 호출해주는 쪽에서 issue가 있을때만 호출해줄 수 있도록 수정하는건 어떨까요?

위에서 작성해주신 이 코멘트랑도 연관된 부분이어서, 아래와 같이 수정해보았습니다.
1. `IssueVC`의 `issue` 프로퍼티를 non-optional하게 수정
  - 새로운 issue 등록을 위해 `IssueVC`에 진입시에는 이니셜라이저에서 새로운 빈 issue를 생성하도록 변경
2. 기존 issue와 새로운 issue를 구분하는 조건을 `issue == nil`이 아닌 `isEditable`로 변경